rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -45,11 +41,7 @@ def _consume_rotki_trades(
         amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
         location = Location.deserialize(csv_row['Location'])
         trade = Trade(
-            timestamp=deserialize_timestamp_from_date(
-                date=csv_row['Timestamp'],
-                formatstr=timestamp_format,
-                location='Rotki generic trades importer',
-            ),
+            timestamp=deserialize_timestamp(timestamp=csv_row['Timestamp']),
```

Comment:
In my opinion it might be a bit confusing if users have to use `milliseconds` timestamp when importing events and `seconds` timestamp when importing trades. What do you think?

Reply:
Yes. It should be milliseconds everywhere and we can convert to seconds internally where needed (though we may also need to eventually switch to ms everywhere at some point in the future)

Reply:
👍